staging/src/k8s.io/apiserver/pkg/registry/rest/resttest/resttest.go

```diff
@@ -835,7 +835,7 @@ func (t *Tester) testDeleteNoGraceful(obj runtime.Object, createFn CreateFunc, g
 	}
 	defer t.delete(ctx, foo)
 	objectMeta := t.getObjectMetaOrFail(foo)
-	opts := metav1.NewDeleteOptions(10)
+	opts := metav1.NewDeleteOptions(0)
```

Comment:
It looked strange that the dryrun call should delete non-graceful although a grace period is set in the options ... What should be the correct behaviour for Pods and CRs?